src/btree/bt_split.c

```diff
@@ -1480,8 +1488,8 @@ __wt_split_multi(WT_SESSION_IMPL *session, WT_REF *ref, int exclusive)
 		    page, &mod->mod_multi[i], &ref_new[i], &parent_incr));
 
 	/* Split into the parent. */
-	WT_ERR(__split_parent(
-	    session, ref, ref_new, new_entries, parent_incr, exclusive));
+	WT_ERR(__split_parent( session, ref, ref_new,
+	    new_entries, parent_incr, (exclusive ? WT_EXCLUSIVE : 0)));
```

Comment:
There is an extra set of parentheses here than necessary - it's best if you take them out. Similarly below.